features/affordability.feature

```diff
@@ -172,3 +172,25 @@ Scenario: Adjusting the mortgage term via the slider
   When  I enter all details for single applicant
   Then  I should see the correct results for a single applicant
   And   I should be able to tweak the results by adjusting the mortgage term slider
+
+Scenario: Viewing next steps for medium risk
+  Given I visit the affordability page
+  When  I enter all details for single applicant at medium risk
```

Comment:
This would still be part of the Given steps I think. the expectation isn't until you visit the next page. 
`And  I enter all details for single applicant at medium risk`

Reply:
Do these tests need to cover both javascript and non-javascript?

Reply:
Hey @willhall88. I thought that as entering the details in such a way that the risk level will come out as low, medium or high is the key action that determines what content should eventually be shown, that would be the `When` part of the scenario? Happy to change them to `And`s though if you prefer.

Reply:
ok. that makes sense. I'd then change the below step to an `And` so it isn't `When... When`
not really a blocker. just me being pedantic 🙈 

The bigger question is if the logic differs between javascript and non-javascript and if both paths need to be tested. I have a feeling we have come across that before in mortgage_calculator with it being a rails/angular monster

Reply:
Ok 👍  will change to a `When` and then an `And`

Reply:
The logic doesn't differ between js/non-js for this bit of work, it's done with different partials for each risk level that are shown based on the risk level on the `Affordability` model, so I think we should be ok?